core/src/main/resources/error/error-classes.json

```
@@ -774,6 +774,12 @@
       "The expected format is ByteString, but was <unsupported> (<class>)."
     ]
   },
+  "INVALID_COLUMN_NAME" : {
```

How about to make it more specific:
```suggestion
  "INVALID_COLUMN_NAME_AS_PATH" : {
```